XavierGuertin/WildfireVisualizationProject
# #34 - fix(frontend): Modified configurations to make builds work locally

### Summary
This pull request adds some necessary config modifications in order to start the CI configuration. Before, the builds were not able to be built since the directory paths were incorrect in the config files.

### Changes Made
- Changed baseConfig in eslint.config.js in the frontend e2e
- Changed baseConfig in eslint.config.js in the frontend
- Changed rootDir in tsconfig.base.json to match the root

### Testing Instructions
If you see the GitHub action run for the CI file is passing, then it is working as intended.
